datadog_checks_base/datadog_checks/base/stubs/aggregator.py

```diff
@@ -418,7 +418,7 @@ def assert_no_duplicate_metrics(self):
         - hostname
         """
         # metric types that intended to be called multiple times are ignored
-        ignored_types = [self.COUNT, self.MONOTONIC_COUNT, self.COUNTER]
```

Comment:
Double submission of a `monotonic_count` is allowed and has a defined behaviour, however it's easily a source of error. While there is no integration submitting a monotonic_count twice on purpose, let's raise an error on these cases.